lib/Command/Import.php

```diff
@@ -48,6 +48,12 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 		$io = new SymfonyStyle($input, $output);
 
 		try {
+			$path = $input->getArgument('archive');
+			if (!file_exists($path)) {
```

Comment:
Will this work with S3 as primary storage?

Reply:
It’s a local zip file that we are importing from.